### ISSUE TITLE
Blank output of transient analysis without warning/error meassage.

### ISSUE THREAD

**Issue by theogam02** [tomcl/ADDIE#16]
There is a type of circuit that will not give you any warning messages when trying to do transient analysis, but the output is _blank._

<img width="430" alt="image" src="https://github.com/tomcl/ADDIE/assets/94688075/530cf65c-6cfb-4b62-83ac-d5a65b8bea12">

If this is due to the simulator not supporting this type of circuit, there should be some sort of error message.

I tried circuits with 1 diode or 1 capacitor and the analysis works. It seems to break down when both a capacitor and a diode are used in a circuit.

When trying a diode and an inductor, I got an error message:

<img width="416" alt="image" src="https://github.com/tomcl/ADDIE/assets/94688075/3cc49b75-a7a2-4e27-a20a-a3be59c9431b">

As stated in the error message "Time Simulation currently supports a maximum of one Voltage Source and one Capacitor or Inductor". This circuit has an inductor and a diode and it generates that error message.

I believe that more than one capacitor/inductor/diode or any combination of those will break the simulator. The error message appears in some of those cases but it does not appear in some others (like the one in the first picture).

 - OS: MacOs 13.2
 - Version 1.0.0

**Comment by tomcl** [comment]
Agreed. I don't understand why time simulation should be so restricted - you would have though any collection of linear components would be OK. I suggest in resolving this issue we should:

1. Add an error checker
2. Understand the component limits on time simulation - make suitable error messages
3. Allow more diverse ckts for time simulation. @apantelopoulos will maybe have some idea what were the technical constraints he came across.
